lib/omniauth/strategies/paypal_permissions.rb

```diff
@@ -110,7 +110,15 @@ def load_identity
           identity["display_name"] = identity["company_name"] || identity["full_name"]
           identity
         else
-          fail!(:invalid_credentials)
+          error = response.error.first
+          if error.present?
+            {
```

Comment:
Thank you for reviewing. It makes sence.

Or, PayPal SDK returns following exception class,  
how about `fail!(:invalid_credentials, response.error.first)`  ?

```
> response.error
=> [#<PayPal::SDK::Permissions::DataTypes::ErrorData:0x007fcbc0faec98
  @category="Application",
  @domain="PLATFORM",
  @errorId=570058,
  @message="Authentication failed. User account is not confirmed.",
  @severity="Error",
  @subdomain="Application">]
```